x/links/types/expected_keeper_ibc.go

```diff
@@ -0,0 +1,21 @@
+package types
+
```

Comment:
I think you don't need to rewrite these interfaces, you can just import the cosmos ones where you need them with this import:
```go
import (
ibcTypes "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
)
```